tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -240,10 +242,16 @@ def test_cluster_ping_and_is_up(self, cluster):
 
         original_ips = cluster.ips
 
-        cluster._cluster_status = ClusterStatus.TERMINATED
+        if cluster.launcher == LauncherType.DEN:
+            cluster._cluster_status = ClusterStatus.TERMINATED
```

Comment:
I think `_ping` would still work fine since the ips are set and reachable through ssh, but `is_up` is the one that explicitly checks against cluster terminated

Reply:
Oh yeah, good point!